ansible/roles_ocp_workloads/ocp4_workload_performance_monitoring/files/sonatype-nexus.yml

```diff
@@ -0,0 +1,94 @@
+kind: NexusRepo
+apiVersion: sonatype.com/v1alpha1
+metadata:
+  name: nexus
+  namespace: shared-maven-mirror
+spec:
+  config:
+    data: null
+    enabled: false
+    mountPath: /sonatype-nexus-conf
+  deployment:
+    additionalContainers: null
+    additionalVolumeMounts: null
+    additionalVolumes: null
+    annotations: {}
+    initContainers: null
+    postStart:
+      command: null
+    preStart:
+      command: null
+    terminationGracePeriodSeconds: 120
+  deploymentStrategy:
+    type: Recreate
+  ingress:
+    annotations: {}
+    enabled: false
+    path: /
+    rules: null
+    tls:
+      enabled: true
+      secretName: nexus-tls
+  nexus:
+    dockerPort: 5003
```

Comment:
Will this need to be opened?

Reply:
@rafaeltuelho

Reply:
No, this is the default config. But the docker port/registry will not be exposed. You can ignore it.